themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1365,6 +1365,10 @@ html body.tc-body.tc-single-tiddler-window {
 	margin-top: 8px;
 }
 
+.tc-tiddler-edit-frame .tc-dropzone-editor > div > div > .tc-editor-toolbar {
```

Comment:
selectors like this one are very brittle if the DOM structure is changed. And they will cause backwards compatibility problems. So if someone needs to fix a bug in the DOM structure. This CSS will make it backwards incompatible, so it will be rejected

Reply:
Thanks @pmario 

yes if the DOM structure changes, these rules need to be adapted

Reply:
IMO go to the DIV of the template and add a new class eg: `tc-grid` to that element. Then `.tc-tiddler-frame .tc-grid` should be enough to set the CSS reliably. -- No js code and no brittle configurations needed.

Reply:
Hi @pmario, I've done that now